packages/react-components/src/components/DevicePermissions/DomainPermissionsScaffolding.tsx

```diff
@@ -116,19 +117,25 @@ export const DomainPermissionsContainer = (props: DomainPermissionsContainerProp
   const theme = useTheme();
   return (
     <Stack style={{ padding: '2rem', maxWidth: '25.375rem' }} aria-label={strings?.ariaLabel}>
-      <Stack horizontal style={{ paddingBottom: '1rem' }} horizontalAlign={'space-between'}>
+      <Stack
+        styles={iconBannerContainerStyles}
+        horizontal
+        horizontalAlign={'center'}
+        verticalFill
+        tokens={{ childrenGap: '2rem' }}
```

Comment:
extract a file-level constant for `tokens` so you're not passing in a new object each time.